docs/src/circuits.md

```diff
@@ -0,0 +1,76 @@
+# Circuits with QXZoo and YaoQX
+
+In the [introduction](basics.md) tutorial circuits are constructed by explicitly defining each of the gate matrices.
+For larger circuits this can get tedious and it is easier to construct circuits using QXZoo or to convert circuits created using [Yao.jl](https://yaoquantum.org/) to TensorNetworkCircuits.
+
+## QXZoo
+QXZoo was developed to simplify this process and provides functions for constructing commonly used circuits as well as primatives for building circuits from commonly used gate sets.
+QXTools wraps many of the functions from QXZoo to improve integration and provides a simple interface for creating [Greenberger–Horne–Zeilinger](https://arxiv.org/abs/0712.0921) (GHZ), Quantum Fourier Transform (QFT) and Random Quantum Circuits (RQC).
+
+To create a GHZ circuit with 5 qubits one would use
+
+```
+using QXTools
+circ = create_ghz_circuit(5)
+```
+This returns a QXZoo Circ struct which contains `num_qubits` and `circ_ops` data members giving the number of qubits and access to the gates respectively.
+QXTools provides `gate_qubits` and `gate_matrix` functions to simplify extracting the qubits that the gate acts upon and the matrix representation of the gate.
+
+```
+println("Circuit has $(circ.num_qubits) qubits")
+for g in circ.circ_ops
+    println("Gate acts on qubits $(gate_qubits(g))")
+    println("Matrix form $(gate_matrix(g))")
+end
+```
+
+QXTools provides the `convert_to_tnc` function which will convert a QXZoo circuit to a TensorNetworkCircuit
+
+```
+tnc = convert_to_tnc(circ)
+```
+
+The `create_qft_circuit` will create a QFT circuit when given the number of qubits.
+
+```
+circ = create_qft_circuit(5)
+```
+
+Finally the `create_rqc_circuit` creates a RQC circuit and takes arguments specifying the number of rows, columns, depth and the seed to use when sampling unitary gates. One can also specify whether or not to add a final layer of Hadamard which is sometimes ommitted.
+
+```
+circ = create_qft_circuit(4, 4, 16, seed=42, final_h=true)
```

Comment:
```suggestion
circ = create_rqc_circuit(4, 4, 16, 42, final_h=true)
```

The seed argument at the moment is a positional argument but I think it would make sense as a keyword argument